sirepo/template/radia.py

```diff
@@ -70,14 +119,24 @@ def background_percent_complete(report, run_dir, is_running):
     )
 
 
+def create_archive(sim):
```

Comment:
shouldn't this just be in Exporter?

Reply:
Yes, but every simulation depends on the existing location of ```create_archive()``` so I want to be sure a) this is the correct approach and b) the radia case is complete